.devcontainer/Dockerfile

```diff
@@ -96,13 +99,34 @@ RUN apt-get update -q && add-apt-repository universe && \
     ros-${ROS_DISTRO}-diagnostic-updater \
     ros-${ROS_DISTRO}-robot-localization \
     ros-${ROS_DISTRO}-common-interfaces \
-    ros-${ROS_DISTRO}-rmw-cyclonedds-cpp && \
-    rm -rf /var/lib/apt/lists/*
+    ros-${ROS_DISTRO}-rmw-cyclonedds-cpp \
+    ros-${ROS_DISTRO}-rosbag2-storage-mcap 
+    # ros-${ROS_DISTRO}-image-transport-plugin \
+    # ros-${ROS_DISTRO}-compressed-image_transport \
+    # ros-${ROS_DISTRO}-compressed-depth-image-transport \
+    # ros-${ROS_DISTRO}-theora-image-transport \
+    # ros-${ROS_DISTRO}-ffmpeg-image-transport \
+    # ros-${ROS_DISTRO}-ffmpeg-encoder-decoder \
+    # ros-${ROS_DISTRO}-point-cloud-transport-plugins \
+    # ros-${ROS_DISTRO}-draco-point-cloud-transport \
+    # ros-${ROS_DISTRO}-zlib-point-cloud-transport \
+    # ros-${ROS_DISTRO}-zed-msgs \
+    # ros-${ROS_DISTRO}-cob-srvs && \
+    # rm -rf /var/lib/apt/lists/*
 
 ENV RMW_IMPLEMENTATION=rmw_cyclonedds_cpp
-
 RUN rosdep init && rosdep update --rosdistro $ROS_DISTRO
 
+###########################
+# Install pip requirements
+###########################
+ENV VIRTUAL_ENV=/root/venv
+RUN python3 -m venv $VIRTUAL_ENV --system-site-packages
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+ENV PYTHYONPATH="$VIRTUAL_ENV/lib/python3.10/site-packages:$PYTHONPATH"
+COPY requirements.txt .
+RUN --mount=type=cache,target=/root/.cache/ pip install -r requirements.txt
+
```

Comment:
Are we missing python version? 
Could we comment the lines for requirements.txt (127 ad 128) as we will use pyproject.toml to track and update dependencies.

Reply:
Need to install uv and then pyproject.toml